Onyimatics/Banka
# #166104176 Add functionality to admin can view specific account details using fetch API

#### Questions:### What does this PR do?
- Add functionality to admin can view specific account details with the transactions under the account using fetch API

#### Description of Task to be completed?
- Add functionality to admin can view specific account details using fetch API
- Add functionality to admin can view the transactions under a specific account details using fetch API

#### How should this be manually tested?
- open [Banka App](https://bankaapp.herokuapp.com)
- signin with email: ezikeonyinyefavour@gmail.com, password: 123456789
- you should be able to view a list of all accounts on a successful signin.
- then click on view button on each of the accounts

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#166104176](https://www.pivotaltracker.com/story/show/166104176)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
